common/network-types/src/session/state.rs

```diff
@@ -771,14 +775,18 @@ mod tests {
     use futures::io::{AsyncReadExt, AsyncWriteExt};
     use futures::pin_mut;
     use futures::stream::BoxStream;
+    use hex_literal::hex;
     use parameterized::parameterized;
-    use rand::{thread_rng, Rng};
+    use rand::rngs::StdRng;
+    use rand::{Rng, SeedableRng};
     use std::fmt::Debug;
     use std::iter::Extend;
     use test_log::test;
 
     const MTU: usize = 466; // MTU used by HOPR
 
+    const RNG_SEED: [u8; 32] = hex!("d8a471f1c20490a3442b96fdde9d1807428096e1601b0cef0eea7e6d44a24c01");
```

Comment:
Could you document the reason for a static seed here?

Reply:
Done